guides/doc-Release_Notes/topics/katello-4.10.0.adoc

```diff
@@ -0,0 +1,157 @@
+= Katello 4.10.0
+
+A full list of changes is available on https://projects.theforeman.org/issues?set_filter=1&sort=id%3Adesc&status_id=closed&f%5B%5D=cf_12&op%5Bcf_12%5D=%3D&v%5Bcf_12%5D%5B%5D=1687[Redmine]
+
+== Katello
+
+* Incorrect aria-label and component id in the alternate content source modal for editing credentials certificates - https://projects.theforeman.org/issues/36512[#36512]
+* ID for activation keys on register host page is incorrect - https://projects.theforeman.org/issues/36489[#36489]
+
+=== API
+
+* Remove deprecated docker_tags_whitelist - https://projects.theforeman.org/issues/36695[#36695]
+* Remove katello-agent API endpoints - https://projects.theforeman.org/issues/36665[#36665]
+
+=== Activation Key
+
+* Add activation key details top bar - https://projects.theforeman.org/issues/36576[#36576]
+* Environment and Content View info is not visible on the Associations Content Host page for Activation keys - https://projects.theforeman.org/issues/36501[#36501]
+
+=== Alternate Content Sources
+
+* ACS bulk refresh through API silently sanitizes input ids - https://projects.theforeman.org/issues/36634[#36634]
+* Custom ACS path help text is missing file:// - https://projects.theforeman.org/issues/36631[#36631]
+* ACS error message contains duplicate words - https://projects.theforeman.org/issues/36590[#36590]
+* Add option --all to "hammer alternate-content-source refresh" command to refresh all ACS\'s - https://projects.theforeman.org/issues/36589[#36589]
+* Incorrect aria-label and component id in the alternate content source modal for editing Products - https://projects.theforeman.org/issues/36514[#36514]
+* Incorrect aria-label and component id in the alternate content source modal for editing smart proxies - https://projects.theforeman.org/issues/36513[#36513]
+* Incorrect aria-label in the alternate content source drawer - https://projects.theforeman.org/issues/36511[#36511]
+* Incorrect aria-label and component id in the alternate content source modal for editing credentials - https://projects.theforeman.org/issues/36510[#36510]
+* \'Remove orphans\' task fails on DeleteOrphanAlternateContentSources step - https://projects.theforeman.org/issues/36461[#36461]
+* Simplified ACS update fails to remove products if product has any empty URL repos - https://projects.theforeman.org/issues/36221[#36221]
+
+=== Client/Agent
+
+* tasks Actions::Katello::BulkAgentAction without any sub-plans and stuck in running/pending - https://projects.theforeman.org/issues/36528[#36528]
+* Add UI banners warning users about katello-agent removal - https://projects.theforeman.org/issues/36526[#36526]
```

Comment:
These two are irrelevant now that katello-agent is gone. May think about omitting these items (not the code, just this text) just to avoid confusion.